server/helpers/youtube-dl/youtube-dl-wrapper.ts

```diff
@@ -118,7 +167,7 @@ class YoutubeDLWrapper {
     } catch (err) {
       this.guessVideoPathWithExtension(pathWithoutExtension, fileExt)
         .then(path => {
-          logger.debug('Error in youtube-dl import, deleting file %s.', path, { err, ...lTags() })
+          logger.debug(`Error in youtube-dl import, deleting file ${path}.`, { err, ...lTags() })
```

Comment:
Why did you update this line?

Reply:
To better fit with the other' log calls syntax.

Reply:
Reverted.